src/group_theory/subgroup.lean

```diff
@@ -1824,10 +1883,113 @@ set.subset.antisymm
     by { rw sup_eq_closure, apply Inf_le _, dsimp, refl })
   ((sup_eq_closure N H).symm ▸ subset_closure)
 
-end subgroup
+@[to_additive] lemma mod_law_left (A' A B : subgroup G) (hA : A' ≤ A) :
+  (A' : set G) * (A ⊓ B : subgroup G) = A ⊓ (A' * B) :=
```

Comment:
I think this statement would be slightly neater as
```suggestion
  (A' : set G) * (B ⊓ A : subgroup G) = (A' * B) ⊓ A :=
```
which keeps the arguments and operators in the same order

Reply:
You could then call the lemma `mul_inf_assoc` and the one below `inf_mul_assoc`